forest/data.py

```diff
@@ -473,116 +473,6 @@ def search(pressures, pressure, rtol=0.01):
         return np.abs(pressures - pressure) < (rtol * pressure)
 
 
-class UMLoader(object):
```

Comment:
This would have been a good candidate for a separate PR..?

Reply:
Possibly, but I've been trying to nibble away at the tech debt to start to make the code base easier for new developers.

Reply:
Nibble away ... but when it's completely independent then it'd be even quicker to nibble and separate bites. 😋